src/app/extensions.h

```diff
@@ -54,6 +55,8 @@ namespace app {
       Max
     };
 
+    bool isDefaultTheme() const;
+
```

Comment:
I think this shouldn't be needed with the new field in ExtensionInfo.

Reply:
I think it is still necessary, since this function is used to identify `Extension` instances (not just `ExtensionInfo` instances).